Libraries/VgVideo/vgKwaVideoClip.cxx

```diff
@@ -24,7 +24,7 @@
 #include <vsl/vsl_binary_io.h>
 #include <vsl/vsl_stream.h>
 #include <vsl/vsl_vector_io.h>
-#include <vsl/vsl_vector_io.txx>
+#include <vsl/vsl_vector_io.hxx>
```

Comment:
Could you please make these changes in a separate PR?

Reply:
Definitely - didn't intend for this to slip in to this branch.  As we talked about, I don't want to get pulled in to vxl issues (this should be a problem on Linux as well as Windows, so my choice of vxl may be bad).